### ISSUE TITLE
Agent - optional installation as service

### ISSUE THREAD

**Issue by rstinear** [team-rocos/platform#5]
During the agent installation process, installing the agent as a service should be optional for the user (raised from #4)

1. Take this as a flag on the command line during init
2. Provide it as an option in the wizard flow

**Comment by edwinhayes** [comment]
Just to clarify, from my point of view, creating the service unit file is fine, just not enabling it.  In our case,  we'll probably override it with a `/etc/systemd/system` version that includes the original but adds additional deps and configuration such as syslog stuff.

**Comment by rstinear** [comment]
Yes that sounds totally sensible - we'll have the service config ready to go but will only actually enable it with an explicit opt-in from the user.
